src/common/tokens/operations.js

```diff
@@ -11,13 +11,26 @@ const loadTokens = () => async (dispatch, getState) => {
 
 const addToken = contractAddress => async dispatch => {
 	const tokenService = getGlobalContext().tokenService;
-	const tokenInfo = await tokenService.getTokenInfo(contractAddress);
-	await tokenService.addToken(tokenInfo);
+	try {
+		const tokenInfo = await tokenService.getTokenInfo(contractAddress);
+		await tokenService.addToken(tokenInfo);
+	} catch (error) {
+		await dispatch(
+			actions.setTokenError(
+				`Sorry, an error occurred while adding the current token: ${error}`
```

Comment:
@eduardosmoura Can you remove the `${error}` from here. We don't want to show an ugly error message to user. It will be logged to sentry so we can debug it later. Cheers

Reply:
Yes sure @rodrigopavezi